compiler/crates/relay-compiler/src/config.rs

```diff
@@ -845,14 +860,11 @@ struct ConfigFileProject {
     js_module_format: JsModuleFormat,
 }
 
-#[derive(Debug, Serialize, Deserialize)]
-#[serde(deny_unknown_fields)]
-pub struct PersistConfig {
-    /// URL to send a POST request to to persist.
-    pub url: String,
-    /// The document will be in a POST parameter `text`. This map can contain
-    /// additional parameters to send.
-    pub params: FnvIndexMap<String, String>,
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(untagged)]
```

Comment:
I like this `serde` feature, the only problem with it: obscure error messages if both/mixed values are provided.